### PR TITLE
Added arguments in VagrantFIle to specify network name

### PR DESCRIPTION
**Pull request by vnaboychenko**

### CODE REVIEW

**Comment by roman-verchikov** [comment]
Neat, +1

**Comment by realsystem** [comment]
+1

**Comment by andybondar** [comment]
https://github.com/andybondar/CloudFerry/blob/devel/devlab/Vagrantfile#L19-L21

Otherwise --force and --help options don't work

Pull request with fix - https://github.com/MirantisWorkloadMobility/CloudFerry/pull/285

**Comment by vnaboychenko** [comment]
@andybondar can you clarify why do we need --force, --help ?

File: devlab/Vagrantfile [hunk 1]
```diff
@@ -12,6 +12,14 @@ File.foreach(config_file) { |line|
   end
 }
 
+
+# If you want to deploy your vm's in isolated particular network, you should specify environment variable:
+# export virtualbox_network_name=test_net1
+# vagrant up grizzly juno cloudferry
+
+options['virtualbox_network_name'] = ENV['virtualbox_network_name'] unless ENV['virtualbox_network_name'].nil?
```
Comment (roman-verchikov) [inline devlab/Vagrantfile]: Environment variables are usually uppercase, please rename.

**Comment by roman-verchikov** [comment]
+1

**Comment by realsystem** [comment]
+1

**Pull request merged**